modules/doc/content/ncrc/applications/ncrc_ondemand_level4.md

```diff
@@ -0,0 +1,83 @@
+## Submission and Job Information
+
```

Comment:
Lots of extra blank lines in this doco - consider removing some of them.

Reply:
nice! another file we don't need. It too was converted into template `ncrc_ondemand_template.md`. I think I created this file by mistake. But I will fix the excessive lines in the renamed template.